src/main/java/healthy/lifestyle/backend/users/dto/SignupRequestDto.java

```diff
@@ -43,4 +43,6 @@ public class SignupRequestDto {
     private String fullName;
 
     @NotNull private Long countryId;
+
+    private int age;
```

Comment:
Change int to Integer since it may be null. Add a custom validation annotation: if age field is not null, then it must be in range of 5 years to 200 years (see validation directory for examples).

Double-check that you have all the necessary tests for positive and negative cases.

Until you're done, please convert this PR to Draft status.